### PR TITLE
replace: Return map if missing path

### PR DESCRIPTION
**Pull request by qinqon**
When the replace expression is referencing a missing path nmpolicy was failing since it cannot find the keys in the state. This PR create new empty dictionaries to fill in the gaps from the replace expression.

### CODE REVIEW

File: nmpolicy/internal/resolver/replace.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func (r replaceOpVisitor) visitMap(p path, mapToVisit map[string]interface{}) (i
 	}
 	interfaceToVisit, ok := mapToVisit[*p.currentStep.Identity]
 	if !ok {
-		return nil, nil
+		interfaceToVisit = map[string]interface{}{}
```
Comment (rhrazdil) [inline nmpolicy/internal/resolver/replace.go:64]: Could we `return map[string]interface{}{}, nil` ?

Comment (qinqon) [reply nmpolicy/internal/resolver/replace.go:64]: Nope, the recursion will add the next "not found" entrie from the path, we should pass the empty state.

Comment (rhrazdil) [reply nmpolicy/internal/resolver/replace.go:64]: Thanks, understood.
I think we can proceed with this solution then. For the current use-case of setting lldp.enabled, we can assume the lldp map to exist in the current state.
If there's a need to set map that doesn't exist, we can re-iterace on https://github.com/nmstate/nmpolicy/pull/107

File: docs/examples/all-ethernet-up/current.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  name: eth0
+  state: up
+  type: ethernet
+- accept-all-mac-addresses: false
+  lldp:
+    enabled: false
+  mac-address: 52:55:00:D1:56:02
+  name: eth1
+  state: down
+  type: ethernet
```
Comment (rhrazdil) [inline docs/examples/all-ethernet-up/current.yaml:15]: Would it make sense to add for example:

```yaml
- name: veth1
  type: veth
```
to exercise the fix in this PR - having an interface that doesn't have the `lldp` map specified.

**Comment by rhrazdil** [comment]
Thank you
/lgtm

**Review by rhrazdil — APPROVED** [review]

**Review by rhrazdil — APPROVED** [review]

**Pull request merged**